app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php

```diff
@@ -26,31 +26,32 @@
  * create the order on Bolt side through the javascript BoltCheckout.configureProductCheckout process.
  *
  */
-class Bolt_Boltpay_Block_Catalog_Product_Boltpay extends Mage_Core_Block_Template
+class Bolt_Boltpay_Block_Catalog_Product_Boltpay extends Bolt_Boltpay_Block_Checkout_Boltpay
 {
     use Bolt_Boltpay_BoltGlobalTrait;
 
     /**
      * Get Product Tier Price
      * @return mixed
+     * @deprecated
      */
     public function getProductTierPrice()
     {
         return Mage::registry('current_product')->getData('tier_price');
     }
 
     /**
-     * Initiates sets up BoltCheckout config.
+     * Generates BoltCheckout cart configuration for product page checkout
      *
-     * @return string
+     * @return string JSON encoded Bolt cart data containing current product
+     * @deprecated
      */
     public function getCartDataJsForProductPage()
     {
         try {
             $currency = Mage::app()->getStore()->getCurrentCurrencyCode();
 
             $productCheckoutCartItem = [];
-            $totalAmount = (float) 0;
```

Comment:
Was the removal of this line intentional?

Reply:
@adentong 
Yes.  The initialization of this occurs just below here:
https://github.com/BoltApp/bolt-magento1/blob/a77a5febaecd03ec2973513acd591849fd50a557/app/code/community/Bolt/Boltpay/Block/Catalog/Product/Boltpay.php#L75

This line is a useless (unused) piece of code.